packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -50,6 +54,20 @@ export const indexPage =
         <div class="header">
           ${yextLogoWhiteSvg}
           <h1>Pages Development</h1>
+          <h4 class="external-links">
+            <a target="_blank" rel="noopener noreferrer" href="${createYextAccountUrl(
+              accountId,
+              universe
+            )}">
+              Yext Account
+            </a>
+            ${externalLinkSvg}
+            <span style="margin:40px;"></span>
```

Comment:
What's this span for? Just adding a margin before the link? If so, add the margin to the link instead.